frontend/server/src/main/java/com/amazonaws/ml/mms/util/logging/QLogLayout.java

```diff
@@ -14,27 +14,36 @@
 
 import com.amazonaws.ml.mms.metrics.Dimension;
 import com.amazonaws.ml.mms.metrics.Metric;
+import org.apache.log4j.Logger;
 import org.apache.log4j.PatternLayout;
 import org.apache.log4j.spi.LoggingEvent;
 
 public class QLogLayout extends PatternLayout {
+    private static final Logger logger = Logger.getLogger(QLogLayout.class);
 
     @Override
     public String format(LoggingEvent event) {
         Object eventMessage = event.getMessage();
         if (eventMessage == null) {
             return null;
         }
+        String marketPlace = "NONE";
+        try {
+            marketPlace = System.getenv("REALM");
+        } catch (Exception e) {
+            logger.info("REALM env variable not set or could not be accessed");
```

Comment:
We don't need this. We should have either a value or default value for each of the metrics and just use it. We shouldn't log anything here.